spec/services/quarterly_message_integration_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "business_time"
+
+# I really hate putting this test here as it does not correspond
```

Comment:
@jessieay good point on number 2.  I didn't think about that. Is it permissible in a feature test here to create a bunch of employees via factory?  The reminder can still be created via capybara.  Some folks are sticklers for feature tests being 100% point and clicky and not using any behind the scenes help.